scvi/data/_anndata.py

```diff
@@ -839,3 +843,206 @@ def _register_anndata(adata, data_registry_dict: Dict[str, Tuple[str, str]]):
     >>> _register_anndata(adata, data_dict)
     """
     adata.uns["_scvi"]["data_registry"] = data_registry_dict.copy()
+
+
+def view_anndata_setup(source):
+    """Prints setup anndata.
+
+    Parameters:
+    -----------
+    source
+        Either AnnData, path to saved AnnData, path to folder with adata.h5ad,
+        or scvi-setup-dict (adata.uns['_scvi'])
+
+    Examples:
+    ---------
+    >>> view_anndata_setup(adata)
+    >>> view_anndata_setup('saved_model_folder/adata.h5ad')
+    >>> view_anndata_setup('saved_model_folder/')
+    >>> view_anndata_setup(adata.uns['_scvi'])
+    """
+    if isinstance(source, anndata.AnnData):
+        adata = source
+    elif isinstance(source, str):
+        # check if user passed in folder or anndata
+        if source.endswith("h5ad"):
+            path = source
+            adata = anndata.read(path)
+        else:
+            path = os.path.join(source, "adata.h5ad")
+            if os.path.exists(path):
+                adata = anndata.read(path)
+            else:
+                path = os.path.join(source, "attr.pkl")
+                with open(path, "rb") as handle:
+                    adata = None
+                    setup_dict = pickle.load(handle)["scvi_setup_dict_"]
+    elif isinstance(source, dict):
+        adata = None
+        setup_dict = source
+    else:
+        raise ValueError(
+            "Invalid source passed in. Must be either AnnData, path to saved AnnData, "
+            + "path to folder with adata.h5ad or scvi-setup-dict (adata.uns['_scvi'])"
+        )
+
+    if adata is not None:
+        if "_scvi" not in adata.uns.keys():
+            raise ValueError("Please run setup_anndata() on your adata first.")
+        setup_dict = adata.uns["_scvi"]
+
+    summary_stats = setup_dict["summary_stats"]
+    data_registry = setup_dict["data_registry"]
+    mappings = setup_dict["categorical_mappings"]
+    version = setup_dict["scvi_version"]
+
+    n_cells = summary_stats["n_cells"]
+    n_vars = summary_stats["n_vars"]
+    n_labels = summary_stats["n_labels"]
+    n_batch = summary_stats["n_batch"]
+    n_proteins = summary_stats["n_proteins"]
```

Comment:
is it so bad to just call summary_stats -> sum_stats and use it directly in the data summary dict? extra 5 lines of code here